react/features/token-data/actions.js

```diff
@@ -0,0 +1,51 @@
+/* @flow */
+
+import { getConfigParamsFromUrl } from '../base/config';
+
+import { SET_TOKEN_DATA } from './actionTypes';
+import TokenData from './TokenData';
+
+/**
+ * Parses token data from query string and if it's found stores into Redux
+ * store.
+ *
+ * @returns {Function}
+ */
+export function parseTokenData() {
+    return (dispatch: Dispatch<*>, getState: Function) => {
+        const state = getState();
+        const { config } = state['features/base/lib-jitsi-meet'];
+        const { roomUrl } = state['features/app'];
+        const params = getConfigParamsFromUrl({
+            dontParse: true,
+            source: 'search',
+            url: roomUrl
+        });
+        const token = params.jwt;
+        const parsedToken = new TokenData(config, token);
+        const { payload } = parsedToken;
+
+        // Use JWT param as token if there is not other token set and if
+        // the iss field is not anonymous. If you want to pass data with
+        // JWT token but you don't want to pass the JWT token for
+        // verification the iss field should be set to "anonymous".
+        if (!config.token && payload && payload.iss !== 'anonymous') {
+            config.token = token;
```

Comment:
This is not going to work since the state won't be updated in the store and should always be immutable anyway. Also, this won't affect the global `config` which the web uses.